doc/widgets.md

```diff
@@ -47,3 +47,31 @@ SpriteButton(
           pressedSprite: _pressedSpriteButton,
 )
 ```
+
+## FlameSpriteWidget
+
+`FlameSpriteWidget` is a widget used to display [Sprites](https://github.com/flame-engine/flame/blob/master/lib/sprite.dart) inside a widget tree.
+
+How to use it:
+
+```dart
+FlameSpriteWidget(
+        sprite: shieldSprite,
+        center: true,
+),
+
+```
+
+## FlameAnimationWidget
+
+`FlameAnimationWidget` is a widget used to display [Animations](https://github.com/flame-engine/flame/blob/master/lib/animation.dart) inside a widget tree.
+
+How to use it:
+
+```dart
+FlameAnimationWidget(
+        animation: _animation,
+        play: true,
+        center: true,
```

Comment:
same. I assume play means wether to hold the first frame still (false) or regularly start playing the animation, but that might not be clear to everyone

Reply:
I think playing will be a better name I guess